tests/tests.js

```diff
@@ -147,7 +147,21 @@ var runTests = function runTests(arrayFrom) {
 		t.end();
 	});
 
-	test('no setters are called for indexes', { 'skip': !Object.defineProperty }, function (t) {
+	var defineProperty = (function () {
```

Comment:
oh wait sorry, you're testing for this specific case. can we name the var something more descriptive than `defineProperty`, and/or include a comment that links to the webkit bug?